templates/test/dev/cluster-template-custom-builds-load.yaml

```diff
@@ -183,12 +282,18 @@ spec:
   template:
     spec:
       identity: UserAssigned
+      image:
+        marketplace:
```

Comment:
The Azure Marketplace images are deprecated and won't be updated. Azure Community Gallery is the new hotness:

```
image:
  computeGallery:
    gallery: "ClusterAPI-f72ceb4f-5159-4c26-a0fe-2ea738f0d019"
    name: "capi-ubun2-2404"
    version: "latest"
```

Reply:
It seems like all the other templates are still using the existing images, I think we can probably follow up in another PR and update them all at once.